src/core/NPDtree.ml

```diff
@@ -38,6 +38,12 @@ let open_term ~stack t = match T.view t with
   | T.App (_, l) ->
     Some {cur_term=t; stack=l::stack;}
 
+let index_as_star t =
+  not (T.is_app t) ||
```

Comment:
what about the `Const` case with atomic type?

Reply:
`T.is_app (Const a) = true`, so `index_as_star (Const a) = false`. I believe this is the correct behaviour.

Reply:
Well spotted. I had forgotten about this.